config/crd/stable/gateway.networking.k8s.io_tcproutes.yaml

```diff
@@ -218,6 +218,84 @@ spec:
                       maxItems: 16
                       minItems: 1
                       type: array
+                    matches:
```

Comment:
Stable CRDs should not change with this PR.

Reply:
things have been moved to `experimental` as you suggested